travis-ci/terraform-config
# Add rules to log and drop spoofed packets

## What is the problem that this PR is trying to fix?
Allowing spoofed packets out into the internet is how we, collectively, as a species, get ants.

## What approach did you choose and why?
Drop packets that are coming into the local network's interface that aren't coming from the local subnet.

## How can you test this?
Deploy it and see if anything works, and then try spoofin' some packets! In fact, better yet, try spoofin' some packets now and see if they go through. (Though, that might be hard, because, well... spoofing.)

## What feedback would you like, if any?
Is this necessary? Is this good? Do you like how I did it?
